apps/web/src/server/applications/create/case/applications-create-case-session.service.js

```diff
@@ -32,3 +33,33 @@ export const getSessionCaseSectorName = (session) => {
 export const destroySessionCaseSectorName = (session) => {
 	delete session.caseSectorName;
 };
+
+/**
+ * Save in the session whether is the first time filling the form or it has been resumed.
+ *
+ * @param {SessionWithCaseHasNeverBeenResumed} session
+ * @returns {void}
+ */
+export const setSessionCaseHasNeverBeenResumed = (session) => {
+	session.caseHasNeverBeenResumed = true;
```

Comment:
this value should be an array of objects with each object like `{id: caseId, neverResumed: true/false}`. this way you can manage the state of each case in Draft. This applies also when you save the checkboxes.